internal/core/cluster/sbd.go

```diff
@@ -54,9 +53,8 @@ type SBDNode struct {
 	Status string
 }
 
-func NewSBD(executor utils.CommandExecutor, cluster, sbdPath, sbdConfigPath string) (SBD, error) {
+func NewSBD(executor utils.CommandExecutor, sbdPath, sbdConfigPath string) (SBD, error) {
 	var s = SBD{
-		cluster: cluster,
```

Comment:
It is not used at all.
I could keep it and exclude it's comparison in the assert, but I think removing it is better.